zuora-config.test.js

```diff
@@ -19,6 +19,11 @@ describe('zuora-config', function() {
     config = require('./zoura-config');
   });
 
+  // clean up state setup
```

Comment:
This needs to happen because state is carried forward from 'zuora-config' in between mocha tests